roles/jd-client/src/lib/job_declarator/message_handler.rs

```diff
@@ -55,12 +55,20 @@ impl ParseServerJobDeclarationMessages for JobDeclarator {
     ) -> Result<SendTo, Error> {
         let tx_list = self
             .last_declare_mining_jobs_sent
-            .get(&message.request_id)
-            .unwrap()
-            .clone()
-            .unwrap()
-            .tx_list
-            .into_inner();
+            .iter()
```

Comment:
I don't get why we should iter over the keys rather then use get

Reply:
btw not really important as last_declare_mining_jobs_sent is supposed to be very small

Reply:
a ok you changed the definition of itignore my prev comments